reporting/sending_metrics_report.md

```diff
@@ -0,0 +1,19 @@
+# Sending Metrics Report
+
+## Configurations
+
+Sending Metrics Report is an automated task in UgandaEMR, however it requires setting some facility Credentials ie. Analytics Username and Password and DHIS2 Facility UUID
+
+To set the Analytics User name and password,
+
+1. Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;UgandaemrSync.
```

Comment:
@dbaluku follow the style of the other steps

1. Go to the Legacy Administration 
2. Click Settings 

>> add screenshot 

3. Select Ugandaemrsync

>> add screenshot

Reply:
Done